### PR TITLE
BUGFIX: Doctrine controller should allow empty host for database

### PR DESCRIPTION
**Pull request by kitsunet**
The doctrine controller checked for correct configuration of a database
before calling the doctrine methods. As the drivers all have different
options the ``host`` is a not a conclusive check. The only thing that
can be checked is if a driver was set.

### CODE REVIEW

**Comment by kitsunet** [comment]
@robertlemke @kdambekalns This should fix the configuration problem regarding sockets and/or no host.

IMHO that makes much more sense for now. Please quickly check. Behavior is as expected.

**Comment by kdambekalns** [comment]
What happens when no host is configured, though? Does some default come from us, Doctrine or even MySQL itself?

**Comment by kitsunet** [comment]
@kdambekalns yes the EntityManagerFactory currently defaults that to 127.0.0.1 as was before via settings

**Comment by kdambekalns** [comment]
I see. Well, I like. :)

File: TYPO3.Flow/Classes/TYPO3/Flow/Command/DoctrineCommandController.php [hunk 1]
```diff
@@ -499,4 +485,14 @@ protected function handleException(\Exception $exception)
         $this->systemLogger->logException($exception);
         $this->quit(1);
     }
+
+    protected function isDatabaseConfigured()
```
Comment (robertlemke) [inline TYPO3.Flow/Classes/TYPO3/Flow/Command/DoctrineCommandController.php]: lacks a DocComment

**Comment by kitsunet** [comment]
Tests pass, will merge for now :)

**Pull request merged**